client/templates/custom-components/letter-flow/redirect.njk

```diff
@@ -1,13 +1,29 @@
-{% extends "../../layouts/default-no-nav.njk" %}
+{% extends "layouts/default-no-nav.njk" %}
 
-{% from "govuk/components/radios/macro.njk" import govukRadios %}
 {% from "govuk/components/button/macro.njk" import govukButton %}
 
 {% block page_title %}{{ serviceName}} - {{ serviceTitle }}{% endblock %}
 {% block page_identifier %}{{ pageIdentifier }} - Redirect to print flow{% endblock %}
 
 {% set currentApp = currentApp %}
 
+{% block content %}
+  <div id="refreshForm" class="js-hidden">
+    <p class="govuk-body">Enable pop ups for being able to see and print this report.</p class="govu-body">
```

Comment:
Not incorrect, but better(?) idiomatic English to say 'Enable pop ups to be able to see and print this report'. Unless Liam gave you this, in which case this is correct Scots, probably.

Reply:
he did not 🤣 I'll update